tox.ini

```diff
@@ -30,7 +30,7 @@ setenv =
   AWS_SQS_ACCESS_KEY_ID=fizz
   AWS_SQS_SECRET_ACCESS_KEY=buzz
 commands =
-  coverage run {toxinidir}/cloudigrade/manage.py test account analyzer dj_auth util
+  coverage run {toxinidir}/cloudigrade/manage.py test {posargs: account analyzer dj_auth util}
```

Comment:
What does adding `{posargs}` here get us? I've never used it so would love to know!

Reply:
This allows you pass in a test case you want to run.  For example `tox -e py36 -- account.tests.test_tasks.AccountCeleryTaskTest.test_persist_inspect_results_aws_cloud_image_not_found` will only run that test.  Must quicker debug cycle when developing new tests.

Reply:
That's a neat feature! I've never seen it done like that before either.